pkg/Microsoft.ML.HalLearners/Microsoft.ML.HalLearners.nupkgproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET additional learners making use of hardware acceleration. They depend on like MlNetMklDeps. </PackageDescription>
```

Comment:
>They depend on like MlNetMklDeps [](start = 88, length = 32)

hmm? Either this is a typo, or I need to, like, change my expectations about package descriptions :)

Reply:
thanks for spotting it. 

---
In reply to: [206716498](https://github.com/dotnet/machinelearning/pull/611#discussion_r206716498) [](ancestors = 206716498)

Reply:
why it's still NuGetpackage?

---
In reply to: [206717007](https://github.com/dotnet/machinelearning/pull/611#discussion_r206717007) [](ancestors = 206717007,206716498)